rbi/sorbet/t.rbi

```diff
@@ -65,6 +65,9 @@ module T
 
   sig {params(value: BasicObject).returns(T.noreturn)}
   def self.absurd(value); end
+
+  sig {params(klass: String).returns(T::Private::LazilyResolvedConstant)}
```

Comment:
having `T::Private::LazilyResolvedConstant` as value that can flow around will make our life much harder to analyze them as they float